assets/airports/lrop.json

```diff
@@ -0,0 +1,1370 @@
+        "OP662": ["N44d33m34.496", "E26d00m36.476"],
+        "OP663": ["N44d33m42.936", "E25d53m33.041"],
+        "OP664": ["N44d33m02.339", "E25d53m39.201"],
+        "OP665": ["N44d33m19.188", "E25d48m28.348"],
+        "OP666": ["N44d32m31.096", "E25d46m59.069"],
+        "OP667": ["N44d45m11.509", "E25d58m51.759"],
+        "OP881": ["N44d35m05.531", "E26d11m36.734"],
+        "OP882": ["N44d34m24.940", "E26d11m42.684"],
+        "OP883": ["N44d35m42.748", "E26d19m57.990"],
+        "OP884": ["N44d35m02.161", "E26d20m03.842"],
+        "OP885": ["N44d37m04.432", "E26d38m48.993"],
+        "OP886": ["N44d36m19.747", "E26d37m56.725"],
+        "OP887": ["N44d37m54.519", "E26d50m45.222"],
+        "OP888": ["N44d37m06.677", "E26d49m05.656"],
+        "OP889": ["N44d21m59.577", "E26d21m56.214"],
+        "OP890": ["N44d46m02.123", "E26d10m00.159"],
+        "OP891": ["N44d49m16.707", "E26d02m38.662"],
+        "OPE"  : ["N44d34m29.210", "E26d12m40.520"],
+        "OPT"  : ["N44d35m32.620", "E26d33m36.560"],
+        "OPW"  : ["N44d33m27.880", "E25d59m04.680"],
+        "ORTIP": ["N43d58m40.000", "E25d19m59.001"],
+        "OSTAL": ["N44d16m40.502", "E26d46m22.377"],
+        "OTL"  : ["N44d34m12.920", "E26d00m02.220"],
+        "OTR"  : ["N44d35m16.770", "E26d14m07.330"],
+        "PELUR": ["N45d17m12.278", "E25d29m17.678"],
+        "PEMOK": ["N44d21m52.891", "E24d20m54.470"],
+        "POGAV": ["N46d46m54.001", "E28d10m00.001"],
+        "POLUN": ["N44d14m14.722", "E25d13m24.144"],
+        "RARIT": ["N44d36m04.377", "E26d24m53.323"],
+        "RASUB": ["N43d58m44.000", "E25d25m05.998"],
+        "RIPGA": ["N45d11m30.998", "E27d54m06.998"],
+        "RIVOS": ["N44d39m20.498", "E27d39m16.999"],
+        "SISGU": ["N44d22m09.891", "E25d27m52.661"],
+        "SOKRU": ["N44d59m34.136", "E25d19m49.048"],
+        "SOMOV": ["N43d42m00.000", "E24d51m00.000"],
+        "SORDU": ["N44d02m33.097", "E25d26m48.226"],
+        "STJ"  : ["N44d55m06.820", "E25d58m37.300"],
+        "TABAV": ["N44d53m22.808", "E26d58m01.012"],
+        "TEVRO": ["N44d20m47.081", "E25d30m07.333"],
+        "TIMUR": ["N43d41m20.000", "E24d14m21.001"],
+        "TOBAG": ["N44d22m37.978", "E26d42m09.949"],
+        "TOSVI": ["N44d45m14.010", "E25d09m40.719"],
+        "TUMOP": ["N44d19m03.599", "E26d17m49.709"],
+        "TUTIX": ["N44d14m24.422", "E25d31m07.611"],
+        "UBOGU": ["N44d45m47.001", "E24d40m05.998"],
+        "URELA": ["N45d29m47.911", "E26d33m39.931"],
+        "USIRO": ["N44d14m30.397", "E25d45m52.599"],
+        "UTIDI": ["N44d15m10.031", "E26d14m01.934"],
+        "UTOKI": ["N44d44m39.242", "E25d51m53.111"],
+        "UVALU": ["N44d42m38.891", "E25d26m38.864"],
+        "VAMON": ["N44d23m57.998", "E24d40m46.999"],
+        "VEVIN": ["N44d42m47.001", "E26d39m06.998"],
+        "VIKBI": ["N44d15m30.999", "E24d39m20.998"]
+    },
+    "airways": {
+        "L5":   ["UBOGU", "TOSVI", "VEVIN", "NETUL", "RIVOS", "CND"],
```

Comment:
Please align colon

Reply:
fixed the colon alignment

The 167 files could be from this commint which is actually a merger from the dev branch 

commit 889e425ab7e68e75d86b4c116768dc908edb5d6e
Author: George Enciu <george.enciu@gmail.com> 2018-02-23 15:19:30
Committer: George Enciu <george.enciu@gmail.com> 2018-02-23 15:19:30
Parent: fa96436683ae9c2cd2aded36500e0df9ebb34e01 (Merge remote-tracking branch 'openscope/develop' into feature/206)
Parent: 0742b35d685f10d4395c219772eefd66d6f49831 (Merge bugfix/895 (#896))
Child: 2733cb1f27c4c706553f226b2055d82a814c0a9b (feature/206 conflict fixing)
Branches: feature/206, openscope/feature/206

Merge remote-tracking branch 'openscope/develop' into feature/206

Reply:
@erikquinn will this be a problem when merging?